api/src/main/resources/liquibase-update-to-2.3.xml

```diff
@@ -33,4 +33,16 @@
     	<renameColumn tableName="allergy" oldColumnName="comment" newColumnName="comments" columnDataType="varchar(1024)"/>
     </changeSet>
 
+    <changeSet id="TRUNK-5413" author="odorajonathan" dbms="mysql">
+      <preConditions onFail="MARK_RAN">
+            <not><columnExists tableName="concept" columnName="category"/></not>
+      </preConditions>
+      <comment>
+          Add Category field to OrderSet
+      </comment>
+      <addColumn tableName="concept">
+            <column name="category" valueNumeric="0"/>
```

Comment:
Have you read about liquibase changesets?

Reply:
I have to an extent, atleast for the basics on what is needed for a changeset for a mapping file. If you could, please specify what I shud prioritise so as to do this part because i think i did for the most part for this line

Reply:
Does your changeset look like the rest of the existing ones?

Reply:
problem comes when trying to add a type, most types i put throw exceptions

Reply:
Are your changes committed?